src/include/homestore/btree/btree_kv.hpp

```diff
@@ -285,6 +285,7 @@ class BtreeTraversalState {
 
     const BtreeKeyRange< K >& input_range() const { return m_input_range; }
     const BtreeKeyRange< K >& working_range() const { return m_working_range; }
+    const BtreeKeyRange< K >& current_range() const { return m_working_range; }
```

Comment:
Why do we need duplicate method. Why can't the current_range be changed to working_range in the caller?

Reply:
@hkadayam you are right. I haven't cleaned it yet. I need to add some more changes, too.